plugins/media-keys/gsd-media-keys-manager.c

```diff
@@ -227,6 +227,11 @@ struct GsdMediaKeysManagerPrivate
         gint             inhibit_suspend_fd;
         gboolean         inhibit_suspend_taken;
 
+        /* UPower stuff */
```

Comment:
Could this “media-keys: Ignore touchpad-toggle if lid is closed” commit be upstreamed too?

Reply:
From the commit message:

```
Ignoring touchpad-toggle events for an extra period after the lid has
    been opened should cover this situation for now. On the longer term we
    need a better understanding of how tablet mode works on these laptops
    and why this event is generated on lid close and when flipping into
    tablet mode.
```

I do not believe this is upstream material; the real problem needs to be fixed at the proper place of the stack.

Reply:
Let's continue this discussion on https://phabricator.endlessm.com/T26457